flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBStateDownloader.java

```diff
@@ -41,8 +43,14 @@
 
 /** Help class for downloading RocksDB state files. */
 public class RocksDBStateDownloader extends RocksDBStateDataTransfer {
-    public RocksDBStateDownloader(int restoringThreadNum) {
+    static final String DOWNLOAD_DURATION_STATE_METRIC = "DownloadStateDurationMs";
```

Comment:
Should we rather keep all (present and future) init metrics key in a dedicated class instead of scattered around?

Reply:
My thinking was that this is a custom RocksDB's metric, so it should be named in the rocksdb module. But on a second thought, moving this name away to `MetricNames` doesn't hurt

Reply:
As a compromise, we could have some RocksDBMetricNames that lives in the Rocks package.

Reply:
I've moved it to the `MetricNames`. This download time could be used by different state backends.